homework/week-0/AllisonBojarski/TextAdvGame(HWproblem#3)/src/nyc/c4q/alizinha/Main.java

```diff
@@ -0,0 +1,26 @@
+package nyc.c4q.alizinha;
+
+public class Main
+{
+
+    public static void main(String[] args)
+    {
+        Room basement = new Room("Basement", "You can barely see your enemy in the shadows.");
+        Weapon knife = new Weapon("scimitar", 10);
+
+        Player player = new Player("Stan", 100, knife, basement);
+        Enemy enemy = new Enemy("Karl", 100, knife, basement);
+
+        if(player.getRoom().equals(enemy.getRoom()))
+        {
+            while(player.getHealthPoints() > 0)
```

Comment:
This fight ultimately ends in the enemy's victory because the enemy attacks first, but I only know that because I can see the code. It would actually still show both characters alive at the end of the loop, I think.

Perhaps some more variations of attacking damage based on different circumstances like shields or dodges. Random number generators can help to leave that up to the player's imagination. Also, some if logic would be good after a damage calculation to break out of the loop and display a different message if one character's health <= 0.